templates/web/base/admin/sitemessage/edit.html

```diff
@@ -11,6 +11,8 @@
 }
 </style>
 
+[% IF body_cobrand.moniker != 'kingston' AND body_cobrand.moniker != 'sutton' %]
+
```

Comment:
This change removes the `<form>` element from the page, so you can't save the message :) Might need to rejig things a bit there!

Also, I wonder if it's worth factoring out the logic here so there's nothing cobrand-specific in the template - maybe add something like `sub wasteworks_only { 1 }` to `Cobrand::KingstonSutton`, which would also make it easy to extend this change to future WW-only cobrands.